tests/test_collection.py

```diff
@@ -132,3 +135,42 @@ def test_collection_management(sys_fabric, client, bad_fabric):
         sys_fabric.delete_collection(col_name)
     assert err.value.error_code == 1203
     assert sys_fabric.delete_collection(col_name, ignore_missing=True) is False
+
+
+def test_insert_from_file(client, tst_fabric_name, col):
+    file = open("files/data.json")
+    documents = json.load(file)
+
+    client._tenant.useFabric(tst_fabric_name)
+    client.insert_document_from_file(
+        collection_name=col.name, filepath="files/data.json"
+    )
+
+    data = client.collection(collection_name=col.name).export(limit=len(documents))
+    entries = ("_id", "_key", "_rev")
+    for doc in data:
+        for key in entries:
+            if key in doc:
+                del doc[key]
+
+    assert documents == data
+    col.truncate()
+
+    client.collection(col.name)
```

Comment:
This line is not needed it seems?

Reply:
@edgargarciamacrometa could you please check this out so we can merge if it is oke? cc @koshy-thomas

Reply:
@dlozina-macrometa I have removed this line, resolved merge conflict and also fixed a test line which was left commented.. You can proceed with the merge

Reply:
@koshy-thomas that is great! Thanks